complex/ft_main.c

```diff
@@ -471,6 +472,8 @@ static void ft_fw_usage(char *program)
 	FT_PRINT_OPTS_USAGE("-x", "exit after test run");
 	fprintf(stderr, "\nClient only options:\n");
 	FT_PRINT_OPTS_USAGE("-f <test_config_file>", "");
+	FT_PRINT_OPTS_USAGE("-a <provider_name>", "");
+	FT_PRINT_OPTS_USAGE("-t <test_name>", "");
```

Comment:
Yeah, for runfabtests.sh we need not use -f for provider. I wanted to use it only here in ubertest and use a different option for specifying the config file name keeping it the same in ubertests and runfabtests.